common/test/acceptance/pages/studio/settings_group_configurations.py

```diff
@@ -167,6 +167,10 @@ def delete_button_is_disabled(self):
         return self.find_css('.actions .delete.is-disabled').present
 
     @property
+    def delete_button_is_present(self):
```

Comment:
Unify to just have a single method-- delete_button_is_present. Then tests that want to check if it is not present can just assert that the value of that is False (as opposed to also having delete_button_is_absent).